mattermost-plugin/server/plugin.go

```diff
@@ -153,6 +152,17 @@ func (p *Plugin) OnActivate() error {
 		return err
 	}
 
+	limits, err := p.API.GetCloudLimits()
+	if err != nil {
+		fmt.Println("ERROR FETCHING CLOUD LIMITS WHEN STARTING THE PLUGIN", err)
```

Comment:
Debug string here, probably, shouldn't be it a logger call?

Reply:
I'm following the pattern of other prints on the method, probably not to rely on the logger before activating the plugin